app/components/team_list/team_list_item/team_list_item.tsx

```diff
@@ -20,6 +20,7 @@ type Props = {
     iconBackgroundColor?: string;
     selectedTeamId?: string;
     onPress: (teamId: string) => void;
+    hideIcon?: boolean;
```

Comment:
Why are we hiding the icon? I could not find any reference to this in the tickets nor a figma attached

Reply:
The new design calls for no icons in the team selector for search, from Figma:

![image](https://github.com/user-attachments/assets/5b34ffc1-1527-4fd0-a404-b90e3ecff92b)